terrawrap/models/graph_entry.py

```diff
@@ -69,65 +68,48 @@ def execute(self, operation: str, debug: bool = False) -> Tuple[int, List[str],
         if debug:
             command_env["TF_LOG"] = "DEBUG"
 
-        # pylint: disable=unused-variable
-        plan_file, plan_file_name = tempfile.mkstemp(suffix=".tfplan")
-
         # We're using --no-resolve-envvars here because we've already resolved the environment variables in
         # the constructor. We are then passing in those environment variables explicitly in the
         # execute_command call below.
         base_args = ["tf", "--no-resolve-envvars", self.abs_path]
         init_args = base_args + ["init"] + self.variables
-        plan_args = (
-            base_args +
-            ["plan", "-detailed-exitcode", "-out=%s" % plan_file_name] +
-            self.variables
-        )
         operation_args = base_args + [operation] + self.variables
 
-        if operation in ["apply", "destroy"]:
-            operation_args += ["-auto-approve"]
-
         init_exit_code, init_stdout = execute_command(
-            init_args, print_output=False, capture_stderr=True, env=command_env
+            init_args,
+            print_output=False,
+            capture_stderr=True,
+            env=command_env,
         )
         if init_exit_code != 0:
             self.state = "Failed"
             return init_exit_code, init_stdout, True
-        if operation in ["apply"]:
-            plan_exit_code, plan_stdout = execute_command(
-                plan_args, print_output=False, capture_stderr=True, env=command_env
-            )
-            operation_args += [plan_file_name]
-        else:
-            plan_exit_code = 0
-            plan_stdout = []
-
-        changes_detected = plan_exit_code != 0
-        if plan_exit_code in (0, 2):
-            self.state = "Success"
-        else:
-            self.state = "Failed"
 
-        if plan_exit_code != 2:
-            return (
-                plan_exit_code,
-                init_stdout + ["\n"] + plan_stdout,
-                changes_detected,
-            )
+        shell = False
+        if operation in ["apply", "destroy"]:
+            operation_args = " ".join(["yes", "yes", "|"] + operation_args)
```

Comment:
Why `yes` twice?

Reply:
Because by default, yes outputs `y`, but Terraform wants `yes`. yes will output whatever the second argument is. So `yes yes` prints `yes`.